src/service/video-manager-impl.js

```diff
@@ -251,10 +261,46 @@ class VideoEntry {
         this.video.pause();
       }
 
+      if (this.autoplayIcon_) {
+        this.vsync_.mutate(() => {
+          this.autoplayIcon_.classList.toggle('amp-video-eq-play',
+            this.isVisible_);
+        });
+      }
     });
   }
 
   /**
+   * Creates a pure CSS animated equalizer icon.
+   * @private
+   * @return {!Element}
+   */
+  createAutoplayIcon_() {
+    const doc = this.ampdoc_.win.document;
+    const icon = doc.createElement('i-amp-video-eq');
+    icon.classList.add('amp-video-eq');
+    // Four columns for the equalizer.
+    for (let i = 1; i <= 4; i++) {
+      const column = doc.createElement('div');
+      column.classList.add('amp-video-eq-col');
+      // Two overlapping filler divs that animate at different rates creating
+      // randomness illusion.
+      for (let j = 1; j <= 2; j++) {
+        const filler = doc.createElement('div');
+        filler.classList.add(`amp-video-eq-${i}-${j}`);
+        column.appendChild(filler);
+      }
+      icon.appendChild(column);
+    }
+    const platform = platformFor(this.ampdoc_.win);
+    if (platform.isSafari() && platform.isIos()) {
+      // iOS Safari can not pause hardware accelerated animations.
+      icon.setAttribute('unpausable', '');
```

Comment:
class would have done too, found the Css easier to understand with it being an attribute